app/_how-tos/install-kgw-securely.md

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
I didn't review this file (other than the broken link) since I think this is just a placeholder based on your PR description.